app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -3,12 +3,16 @@ import { Card, Button, Form, Col } from 'react-bootstrap';
 import moment from 'moment-timezone';
 import { PropTypes } from 'prop-types';
 import * as yup from 'yup';
+import Select from 'react-select';
```

Comment:
Why are we bringing in a new library for this exactly?

Reply:
Still want to hear from @sgober about this, but we may want to use this library in place of datalists everywhere (including jurisdiction path datalists).

Reply:
Yeah I think the answer is because Bootstrap doesn't support an input dropdown with autocomplete without a lot of customization, but I wanted to be sure.

Reply:
@kylietmo you got it.  if you pull master and look at chrome vs safari, they look completely different.  this way it is a consistent look and feel while keeping the functionality we want.  it's also has many more features, for instance we use it as a multi select elsewhere in the app

Reply:
@holmesie I'd recommend pulling this in for anything that is not a trivial couple option single select, but that's not a task for right now

Reply:
Got it, thanks!